models/salesforce__contact_enhanced.sql

```diff
@@ -65,7 +65,7 @@ select
     {% endif %}
 
     {% if var('user_pass_through_columns',[]) != [] %}
-    , {{ var('user_pass_through_columns') | join (", user.")}}
+    , salesforce_user.{{ var('user_pass_through_columns') | join (", salesforce_user.")}}
```

Comment:
Nice work with the changes!

Sorry for the confusion. To make sure we're accounting for all additional potential issues, you will also want to make the same changes for the other joins in [line 58](https://github.com/fivetran/dbt_salesforce/blob/305474f89433e8c4c1e7bec5dffced192abe6caf/models/salesforce__contact_enhanced.sql#L58) (adding `contact.` in front of `{{ var('contact_pass_through_columns')`) and [line 63](https://github.com/fivetran/dbt_salesforce/blob/305474f89433e8c4c1e7bec5dffced192abe6caf/models/salesforce__contact_enhanced.sql#L63) (adding `account.` in front of `{{ var('contact_pass_through_columns')`).

You will want to replicate these changes for `salesforce__opportunity_line_item_enhanced` model as well on the `oli` ([on line 61](https://github.com/fivetran/dbt_salesforce/blob/305474f89433e8c4c1e7bec5dffced192abe6caf/models/salesforce__opportunity_line_item_enhanced.sql#L61)) and `product_2` ([on line 66](https://github.com/fivetran/dbt_salesforce/blob/305474f89433e8c4c1e7bec5dffced192abe6caf/models/salesforce__opportunity_line_item_enhanced.sql#L66)) joins as well!